### DISCUSSION TITLE
Welcome to steam-password-change Discussions!

### DISCUSSION THREAD

**Discussion by sometastycake** [sometastycake/steam-password-change#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sanceca** [comment]
The error "lxml.etree.ParserError: Document is empty" is due to what reason?

**Reply by sometastycake** [reply]
> The error "lxml.etree.ParserError: Document is empty" is due to what reason?

Is it relevant?
